domain/src/main/java/org/oppia/domain/classify/rules/dragAndDropSortInput/DragDropSortInputIsEqualToOrderingWithOneItemAtIncorrectPositionClassifierProvider.kt

```diff
@@ -0,0 +1,41 @@
+package org.oppia.domain.classify.rules.dragAndDropSortInput
+
+import javax.inject.Inject
+import org.oppia.app.model.InteractionObject
+import org.oppia.app.model.ListOfSetsOfHtmlStrings
+import org.oppia.app.model.StringList
+import org.oppia.domain.classify.RuleClassifier
+import org.oppia.domain.classify.rules.GenericRuleClassifier
+import org.oppia.domain.classify.rules.RuleClassifierProvider
+
+/**
+ * Provider for a classifier that determines whether two objects of [ListOfSetsOfHtmlStrings] differ by
+ * exactly ordering of one item in the list.
+ *
+ * https://github.com/oppia/oppia/blob/132b9d8f059253548ea1efadf1ff76416dfa2832/extensions/interactions/DragAndDropSortInput/directives/drag-and-drop-sort-input-rules.service.ts#L72
+ */
+internal class DragDropSortInputIsEqualToOrderingWithOneItemAtIncorrectPositionClassifierProvider @Inject constructor(
```

Comment:
I think these lines are exceeding 100 characters (You can add a line limit at 100 characters on your Android Studio UI to verify and wrap the lines!)

Reply:
okay will do that but keeping 100 characters is a bit difficult with these long names

Reply:
@anandwana001 for this case I have to add a comment below and above this function right?

Reply:
> @anandwana001 for this case I have to add a comment below and above this function right?

it's `/* ktlint-disable max-line-length */` above that line and `/* ktlint-enable max-line-length */` below the line. The line which is exceeding that grey line. This should be applied only where we don't have any possible chance to put things on new line.

Reply:
We going for single line comment because in an earlier version of ktlint, single comment thing not supposed to work but as we are going for a specific version of ktlint, we are good to go to use single line comment thing.